tests/test_advanced_options.py

```diff
@@ -99,3 +101,25 @@ def test_analysis_of_private_repo(analysis_data):
     assert len(violations) > 1, "Expected issues are missing";
 
     manage_credentials_in_maven_xml(custom_maven_settings, True)
+
+def test_no_container_leftovers(analysis_data):
+    application_data = analysis_data['jee_example_app']
+    command = build_analysis_command(
+        application_data['file_name'],
+        application_data['source'],
+        application_data['target']
+    )
+    process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
+
+    time.sleep(10)
+    process.send_signal(signal.SIGINT)
+
+    process.wait()
+
+    leftover = subprocess.run("podman ps | grep -v CONTAINER", shell=True, stdout=subprocess.PIPE, encoding='utf-8')
```

Comment:
Using the `grep` utility will force us to execute the tests on bash, so it will fail on Windows if the tests are executed using other CLI

Reply:
Fixed